pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -279,6 +279,11 @@ func (asw *actualStateOfWorld) MarkVolumeAsAttached(
 	return err
 }
 
+func (asw *actualStateOfWorld) MarkVolumeAsAttachedByUniqueVolumeName(
```

Comment:
@saad-ali @pmorie Does it make sense to have this be public? If we could make MarkVolumeAsAttached smart enough to know when to use AddVolumeByUniqueVolumeName so users of asw should not care that is a much better solution. I think if we expect folks outside of this file to keep track of whether they need a unique name or not it is going to be setting ourselves up for someone to get it wrong.